tst/blackbox_repository/test_comparison.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import pytest
+import numpy as np
+
+from yahpo_gym import BenchmarkSet
+
+from syne_tune.blackbox_repository import load_blackbox
+from syne_tune.blackbox_repository.utils import metrics_for_configuration
+from benchmarking.commons.benchmark_definitions.lcbench import (
+    lcbench_selected_datasets,
+)
+from benchmarking.commons.benchmark_definitions import (
+    yahpo_lcbench_benchmark_definitions,
+    yahpo_fcnet_benchmark_definitions,
+    lcbench_benchmark_definitions,
+    fcnet_benchmark_definitions,
+)
+from syne_tune.config_space import cast_config_values
+
+
+benchmark_pairs_lcbench = [
+    (
+        yahpo_lcbench_benchmark_definitions["yahpo-lcbench-" + dataset],
+        lcbench_benchmark_definitions["lcbench-" + dataset],
+    )
+    for dataset in lcbench_selected_datasets
+]
+
+
+benchmark_pairs_fcnet = [
+    (
+        yahpo_fcnet_benchmark_definitions["yahpo-" + name],
+        benchmark,
+    )
+    for name, benchmark in fcnet_benchmark_definitions.items()
+]
+
+
+benchmark_pairs = benchmark_pairs_fcnet
+
+
+# This test is still WIP
```

Comment:
Can we take this one out given that it is WIP and it would take a long time to run in the pipeline? (We would still have access to it with git)

Reply:
Yes

Reply:
Actually, the test is not run in the pipeline. I'd like to keep it in for lcbench, because once they refit their surrogate, I'd like to do the comparison again.
Eventually, we should run such tests somehwere.

Reply:
If the test is not run in the pipeline and need further work then I would be keen on not having it. We do not loose the code by including it in this PR but we have to maintain some code that is in WIP state if we include it.

Reply:
OK